gitlab/metadata.csv

```diff
@@ -267,8 +267,6 @@ gitlab.go_memstats_sys_bytes,gauge,,byte,,The number of bytes obtained by system
 gitlab.go_threads,gauge,,request,second,The number of OS threads create,0,gitlab,go threads,
 gitlab.http_request_duration_microseconds,gauge,,request,second,The HTTP request latencies in microseconds,0,gitlab,HTTP req latencies,
 gitlab.http_request_size_bytes,gauge,,byte,,The HTTP request sizes in bytes,0,gitlab,HTTP req sizes,
-gitlab.http_requests_total,count,,request,second,[OpenMetrics V1] The total number of HTTP requests made,0,gitlab,total HTTP reqs,
-gitlab.http_requests.count,count,,request,second,[OpenMetrics V2] The total number of HTTP requests made,0,gitlab,total HTTP reqs,
```

Comment:
Nit: Instead of removing these legacy metrics, could we append `[Legacy]` to their descriptions instead? It looks like we do that in [`haproxy`](https://github.com/DataDog/integrations-core/blob/952ba2bcb261518fe99c35a91d5118ce683111f6/haproxy/metadata.csv#L80).

Reply:
Sounds good to me. Updated!